slasher/rpc/server_test.go

```diff
@@ -219,481 +217,3 @@ func TestServer_SameSlotSlashable(t *testing.T) {
 		t.Errorf("Wanted slashing proof: %v got: %v", want, sr.ProposerSlashing[0])
 	}
 }
-
```

Comment:
tests moved to slasher/detection/attestations/detect_surround_surrounded_test.go